metaci/plan/forms.py

```diff
@@ -32,20 +32,22 @@ def __init__(self, plan, repo, user, *args, **kwargs):
         self.advanced_mode = False
         if 'advanced_mode' in args[0]:
             self.advanced_mode = (args[0]['advanced_mode'] == "1")
+        else:
+            self.fields['keep_org'].widget = forms.HiddenInput()
```

Comment:
move this down to point where we actually branch on `if self.advanced_mode` to keep the behavior seperate from where we detect it

Reply:
Done in latest commit